webapp_frontend/src/components/pages/Health.tsx

```diff
@@ -49,24 +69,38 @@ export default function Health() {
                 }}
             />
 
-
             <div>
                 <Table striped bordered hover id={"ff-heath-table"}>
                     <thead>
-                    <tr>
-                        <th>Backend information</th>
-
-                    </tr>
+                        <tr>
+                            <th>System Health</th>
+                        </tr>
                     </thead>
                     <tbody>
-                    <tr>
-                        <td>Uptime</td>
-                        <td>{backendLiveTime}</td>
-                    </tr>
-                    <tr>
-                        <td>Usercount</td>
-                        <td>{backendUserCount}</td>
-                    </tr>
+                        <tr>
+                            <td>Deployment Type</td>
+                            <td>{(systemHealthData === null || systemHealthData.deployment === null) ? errorMsg : systemHealthData.deployment}</td>
+                        </tr>
+                        <tr>
+                            <td>Version</td>
+                            <td>{(systemHealthData === null || systemHealthData.version === null) ? errorMsg : systemHealthData.version}</td>
+                        </tr>
+                        <tr>
```

Comment:
can we like hide the whole table and show a loading animation, only if the request fails show "not reachable"

Reply:
https://react-bootstrap.netlify.app/components/spinners/#spinners

Reply:
done.